### PR TITLE
MySQL database conversion

### PR DESCRIPTION
**Pull request by mbMosman**
Conversion from Postgresql to MySQL. 
Also removes some unused code, comments server-side routes, updates server logs for consistent format and logs WARN on auth errors for restricted features.

### CODE REVIEW

File: database.sql [hunk 1]
```diff
@@ -1,103 +1,61 @@
...
--- You must use double quotes in every query that user is in:
--- ex. SELECT * FROM "user";
--- Otherwise you will have errors!
-CREATE TABLE "user" (
-	"id" serial PRIMARY KEY,
-	"username" varchar(255) NOT NULL UNIQUE,
-	"password" varchar(255) NOT NULL UNIQUE,
-	"auth_level" integer(255)
+-- Create tables
+CREATE TABLE user (
```
Comment (mbMosman) [inline database.sql:5]: Note MySQL is not cool with the "s on the tables, columns, etc.

File: database.sql [hunk 2]
```diff
@@ -1,103 +1,61 @@
...
-	"description" TEXT
+CREATE TABLE dojo (
+	id serial PRIMARY KEY,
+	dojo_name VARCHAR(255) NOT NULL,
+	region_name VARCHAR(255) NOT NULL,
+	image VARCHAR(1000),
+	description TEXT
 );
 
-CREATE TABLE "ranks" (
```
Comment (mbMosman) [inline database.sql:21]: Table was unused. Hard-coded values in the client for drop-downs.

File: database.sql [hunk 3]
```diff
@@ -1,103 +1,61 @@
...
 
-INSERT INTO "user_data" (
```
Comment (mbMosman) [inline database.sql:69]: Moved test data to the `database_test.sql` file and added some additional stuff to round it out for testing.

File: package.json [hunk 4]
```diff
@@ -18,10 +18,10 @@
     "express": "^4.16.4",
     "fontsource-roboto": "^3.0.3",
     "material-ui": "^0.20.2",
-    "moment": "^2.29.0",
+    "moment": "^2.29.1",
```
Comment (mbMosman) [inline package.json:21]: The moment library was added to help with parsing & validating dates from the client.

**Review by emersona-glitch — APPROVED** [review]
Thanks for the opportunity to look over your changes, Mary :-)

**Pull request merged**